### PR TITLE
Pipeline and installer tweaks

### PR DESCRIPTION
**Pull request by daviddekoning**
Hi @peterjgrainger, this is a short PR that makes two changes:

1. the installer now checks the correct file for the Rhino/Grasshopper version (bug we inherited from the original installer)
2. the github action for creating a release now marks it as pre-release. This lets us test the artifact before it starts getting auto downloaded...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ The installer does not require admin privileges and auto updates!
 4. Select the `Master` branch
 5. Input the first three digits of the release version in the input box. e.g. `1.8.32`
 6. Press the `Run Workflow` button
+7. Check the created release, if all is well, press the `Edit` button and until the `Pre-release` checkbox
```
Comment (peterjgrainger) [inline README.md:25]:
```suggestion
7. Download the installer and check the created release
8. If all is well, press the `Edit` button on the release, unselect the `This is a pre-release` checkbox and press `update release`
```

**Review by peterjgrainger — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (daviddekoning) [reply README.md]: thanks, that's much clearer

**Comment by daviddekoning** [comment]
@peterjgrainger any further concerns?

**Review by peterjgrainger — APPROVED** [review]

**Comment by peterjgrainger** [comment]
Nope, all good 😁

**Pull request merged**